platform/security/sts/security-sts-attributequerycommon/src/main/java/org/codice/ddf/security/claims/attributequery/common/AttributeQueryClaimsHandler.java

```diff
@@ -128,7 +129,11 @@ public ProcessedClaimCollection retrieveClaimValues(ClaimCollection claims,
                 }
             }
         } catch (URISyntaxException e) {
-            LOGGER.error(ERROR_RETRIEVING_ATTRIBUTES, externalAttributeStoreUrl, nameId, e);
+            LOGGER.warn(
+                    ERROR_RETRIEVING_ATTRIBUTES + "Set log level to DEBUG for more information.",
```

Comment:
Do we really want to tell an Admin to set the log level to DEBUG? if so I think we should call out the package to set so they don't get overwhelmed with logs.